src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
 export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
 
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
 
+  constructor(
+    public color: 'red' | 'green' | 'blue',
```

Comment:
You are using this type more than once, so it is better to create a separate type for it
example:
`type FigureColor = 'red' | 'green' | 'blue';`